cmd/crates/soroban-spec-tools/src/lib.rs

```diff
@@ -1317,6 +1332,7 @@ impl Spec {
                 enum_.cases.iter().next().map(|c| c.value.to_string())
             }
             Some(ScSpecEntry::FunctionV0(_) | ScSpecEntry::UdtErrorEnumV0(_)) | None => None,
+            Some(ScSpecEntry::EventV0(_)) => todo!("EventV0 is not supported yet"),
```

Comment:
This shuld return `None` like the line above because an event can't be used as a function parameter.